css/styles.css

```diff
@@ -0,0 +1,104 @@
+body {
+  font-family: sans-serif;
+  font-size: 18px;
+  line-height: 1.5;
+  margin: auto;
+  padding: 20px;
+  background: #ffcc33;
+}
+
+a {
+  font-weight: bold;
+  color: #05080C;
+	}
```

Comment:
:hammer: Make sure to keep your closing curly brackets all the way to the left - best practice & good indentation.